LoopKit/Service/Remote/RemoteActionDelegate.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  RemoteActionDelegate.swift
+//  LoopKit
+//
+//  Created by Bill Gestrich on 3/19/23.
+//  Copyright © 2023 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+
+public protocol RemoteActionDelegate: AnyObject {
+    func updateRemoteOverride(name: String, durationTime: TimeInterval?, remoteAddress: String) async throws
```

Comment:
Does this enact an override, or just "update" it? I don't see an "enact" version.

Reply:
Yeah enact probably works better. This is for either starting a new override _or_ updating an existing - no distinction between them here.

I pushed an update for the name.